run_benchmarks.rb

```diff
@@ -323,6 +324,10 @@ def run_benchmarks(ruby:, ruby_description:, categories:, name_filters:, out_pat
     args.out_path = v
   end
 
+  opts.on("--out-exact=OUT_FILE", "write exactly this output file plus file extension, ignoring directories, overwriting if necessaryß") do |v|
```

Comment:
```suggestion
  opts.on("--out-exact=OUT_FILE", "write exactly this output file plus file extension, ignoring directories, overwriting if necessaryß") do |v|
```

There's a stange unicode character that showed up here 😮